.github/workflows/make_release.yml

```diff
@@ -34,6 +39,10 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Create NPM version tag
+        run: |
+          echo "NPM_TAG=$(sed -n -e '1,/^version/p' tfhe/Cargo.toml | grep '^version[[:space:]]*=' | tr -d 'version =')" >> "${GITHUB_ENV}"
```

Comment:
'version =' does not match the grep regex, do you want to go ahead like this ?

Reply:
What do you mean it doesn't match ? The missing repetition of `[[:space:]]` ?

Reply:
that's why we use cut in the Makefile